nexus/src/sagas.rs

```diff
@@ -772,6 +792,35 @@ async fn sdc_regions_ensure_undo(
     Ok(())
 }
 
+async fn sdc_create_volume_record(
+    sagactx: ActionContext<SagaDiskCreate>,
+) -> Result<db::model::Volume, ActionError> {
+    let osagactx = sagactx.user_data();
+
+    let volume_id = sagactx.lookup::<Uuid>("volume_id")?;
+    let volume = db::model::Volume::new(
+        volume_id,
+        // TODO: Patch this up with serialized contents that Crucible can use.
+        "Some Data".to_string(),
```

Comment:
These contents are "obviously wrong", but I wanted to separate DB modifications from the actual plumbing into Crucible.

(I figure the construction provided [here](https://github.com/oxidecomputer/omicron/compare/main...jmpesp:volumes?expand=1#diff-795d5fff67c353d4e842acd3635a7e74dbb3c1182e4477d86d3726340cc9077eR739-R770) could be used - this may require looking up the "allocated regions" info from the prior saga step)

We also likely need an additional saga step to "actually send the request to crucible to create the volume", but that can consume the output of the `sdc_create_volume_record` action.

Reply:
> We also likely need an additional saga step to "actually send the request to crucible to create the volume", but that can consume the output of the sdc_create_volume_record action.

Not necessarily - the actual volume construction only occurs when the disk is attached to a VM.